assets/airports/eidw.json

```diff
@@ -1,1329 +1,5087 @@
-                ["N53.369820100", "W6.261274700"],
-                ["N53.369631900", "W6.260616600"],
-                ["N53.369423400", "W6.259976200"],
-                ["N53.369195100", "W6.259354900"],
-                ["N53.368947700", "W6.258754700"],
-                ["N53.368681900", "W6.258177000"],
-                ["N53.368398400", "W6.257623600"],
-                ["N53.368098100", "W6.257095900"],
-                ["N53.367781800", "W6.256595400"],
-                ["N53.367450400", "W6.256123300"],
-                ["N53.367104600", "W6.255680900"],
-                ["N53.366745600", "W6.255269600"],
-                ["N53.366374300", "W6.254890200"],
-                ["N53.365991800", "W6.254544000"],
-                ["N53.365598900", "W6.254231600"],
-                ["N53.365197100", "W6.253954100"],
-                ["N53.364787000", "W6.253712100"],
-                ["N53.364370100", "W6.253506200"],
-                ["N53.363947400", "W6.253337100"],
-                ["N53.363520000", "W6.253205100"],
-                ["N53.363089000", "W6.253110700"],
-                ["N53.362655900", "W6.253053900"],
-                ["N53.362221400", "W6.253035100"],
-                ["N53.361787000", "W6.253054200"],
-                ["N53.361353800", "W6.253111200"],
-                ["N53.360922800", "W6.253206000"],
-                ["N53.360495600", "W6.253338200"],
-                ["N53.360072800", "W6.253507600"],
-                ["N53.359656000", "W6.253713600"],
-                ["N53.359246000", "W6.253955900"],
-                ["N53.358844100", "W6.254233600"],
-                ["N53.358451400", "W6.254546100"],
-                ["N53.358068900", "W6.254892600"],
-                ["N53.357697700", "W6.255272100"],
-                ["N53.357338800", "W6.255683500"],
-                ["N53.356993100", "W6.256125900"],
-                ["N53.356661700", "W6.256598000"],
-                ["N53.356345500", "W6.257098700"],
-                ["N53.356045300", "W6.257626300"],
-                ["N53.355761900", "W6.258179700"],
-                ["N53.355496200", "W6.258757200"],
-                ["N53.355248900", "W6.259357300"],
-                ["N53.355020700", "W6.259978400"],
-                ["N53.354812200", "W6.260618800"],
-                ["N53.354624100", "W6.261276700"],
-                ["N53.354456800", "W6.261950300"],
-                ["N53.354310800", "W6.262637700"],
-                ["N53.354186600", "W6.263337000"],
-                ["N53.354084500", "W6.264046400"],
-                ["N53.354004700", "W6.264763900"],
-                ["N53.353947600", "W6.265487400"],
-                ["N53.353913300", "W6.266214900"],
-                ["N53.353901800", "W6.266944400"],
-                ["N53.353913300", "W6.267674000"],
-                ["N53.353947600", "W6.268401500"],
-                ["N53.354004700", "W6.269125000"],
-                ["N53.354084500", "W6.269842500"],
-                [
```

Comment:
Additionally, I'm not sure what the purpose is for `_PREOLAPO`, but it seems like a workaround to a feature we don't yet have (perhaps fly-over fixes again?). To the degree practical, we should not be creating workarounds to features that will exist soon anyway.